cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -94,24 +94,14 @@ Additionally:
 - Upon successful creation/deletion of an entity, the privileges remain unaffected.
   It is the responsibility of the administrator to delete privileges from the authorization backend on entity deletion.
   If the privileges are not deleted and the entity is recreated the privileges will affect the enforcement on the entity.
-- More privileges are needed for the following operations in the following cases:
-    - Deploying an application with new artifact requires *ADMIN* on the artifact.
-    - Deploying an application with existing artifact requires *READ, WRITE, EXECUTE,* or *ADMIN* on the artifact.
-    - Deploying an application requires *ADMIN* privilege on all the datasets and streams that will be created
-      along with the app, and *ADMIN* privilege on the dataset module and types if there are custom datasets.
-    - Creating a dataset needs *ADMIN* privilege on the dataset module and types if it is a custom dataset.
-    - If no impersonation is involved, correct privileges on the streams and datasets needs to be given to *cdap* to allow
-      *cdap* accessing these entities.
-    - If impersonation is involved, *admin* privilege on the principal is required to create a namespace, deploy an app
-      create dataset or streams.
-
-CDAP does **not** support hierarchical authorization enforcement, which means that privileges on each entity
-are evaluated independently. CDAP has the concept of visibility, which means user will be able to view the entity if
-the user has any of the privilege on the entity or any of its descendants. For example, user can see the application if the user has
-*ADMIN* on the program. Note that visibility should not be confused with enforcement. In this case the user will not be
-able to to perform *ADMIN* action on the application itself.
-
-Authorization policies for various CDAP operations are listed in these tables:
+- CDAP does **not** support hierarchical authorization enforcement, which means that privileges on each entity
+  are evaluated independently. CDAP has the concept of visibility, which means user will be able to view the entity if
```

Comment:
Talking about visibility here can confuse readers. The only thing that users need to know about visibility is already said in `list or view` and `get` operation sections above, and in the auth matrix below.